.github/workflows/ci.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
         with:
           node-version: ${{ matrix.node }}
       - run: npm ci
-      - run: npx c8 --all --src packages/${{ matrix.package }} -x "packages/!(${{ matrix.package }})" --reporter lcovonly --reporter text wsrun --no-prefix -p barnard59-${{ matrix.package }} -c test
+      - run: npx c8 --all --src packages/${{ matrix.package }} -x "packages/!(${{ matrix.package }})" --reporter lcovonly --reporter text npx wsrun --no-prefix -p barnard59-${{ matrix.package }} -c test
```

Comment:
So we keep `wsrun` here? (same for line 75)

Reply:
I thought you'd comment on that. I thought there would be some issue with `npm run -w` but I was wrong